### ISSUE TITLE
[1.12] Client Crash stuck in rftools dim unable to re-enter game untill manually placed out of dimension

### ISSUE THREAD

**Issue by Morpheus1101** [McJty/RFToolsDimensions#130]
User on the MMD-Dev was stuck in a rftools dim an crashing each time he tried to rejoin untill i manually placed them back into overworld, quick flick through github issues and i dont see anything existing for it and the only information i have form the user currently is: https://puu.sh/xcgAs/ec46ab8b91.txt

**Comment by Morpheus1101** [comment]
https://media.discordapp.net/attachments/208710234175307786/347872331588173824/unknown.png

**Comment by McJty** [comment]
Are you using Sponge?

**Comment by Morpheus1101** [comment]
No, MMD-Dev uses normal forge

**Comment by WhiteZ** [comment]
I'm having a strikingly similar issue, using the latest RFToolsDimensions (5.05) and Forge (1.10.2-12.18.3.2511). Creating a new dimension and entering works just fine, but if I log out inside and try to log back in, a NPE occurs somewhere on the client, locking it up and filling the logs until the process is terminated. The server itself is just fine.

Relevant log: https://gist.github.com/WhiteZ/1a237eba49ac0688c2e7c232e64b7859 (clipped to avoid 2m lines of the same stack trace over and over)

If it can be of any use, here's the dimension information:
```
[18:27:04] [Server thread/INFO]: Dimension ID 68
[18:27:04] [Server thread/INFO]: Description string @Bminecraft:mushroom_island,CSingle,TVoid,MDefault,snormal.day,sdark.night,sdark.purple,sblack.fog,sbody.small.sun,sbody.small.sun,Whard.thunder,tSlow,fNone,fNoDimletBuildings,SNone,eSaturation1,eRegeneration2,d4,d7,d8,d1
[18:27:04] [Server thread/INFO]: Owned by: <snip>
```

**Comment by josephcsible** [comment]
Should be fixed by 024cc63, which will be in the next release.
